docs/guides/creatingasource.md

```diff
@@ -40,6 +43,10 @@ Open the web interface in a browser at the URL found with the following command:
 $ kubectl get ksvc sockeye -o=jsonpath='{.status.url}'
 ```
 
+!!! tip "Minikube"
+    This guide assumes that the Knative service can be addressed from your browser. When using
+    minikube make sure to configure the [Magic DNS option](https://knative.dev/docs/install/yaml-install/serving/install-serving-with-yaml/#configure-dns).
```

Comment:
Is that the case though? What about https://minikube.sigs.k8s.io/docs/commands/service/?

If the concern is about Knative's ingress being sensible to the HOST header, maybe we could simply eliminate all this complexity by updating the guide to use a Kubernetes Service + Deployment instead?

I've had this manifest on my hard drive for longer than I can remember:

```yaml
apiVersion: v1
kind: Service
metadata:
  name: sockeye
spec:
  selector:
    app.kubernetes.io/name: sockeye
  ports:
  - port: 80
    protocol: TCP
    targetPort: 8080
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: sockeye
spec:
  selector:
    matchLabels:
      app.kubernetes.io/name: sockeye
  template:
    metadata:
      labels:
        app.kubernetes.io/name: sockeye
    spec:
      containers:
      - name: sockeye
        image: docker.io/n3wscott/sockeye:v0.7.0@sha256:e603d8494eeacce966e57f8f508e4c4f6bebc71d095e3f5a0a1abaf42c5f0e48
```

```console
$ kubectl port-forward svc/sockeye 8080:80
Forwarding from 127.0.0.1:8080 -> 8080
Forwarding from [::1]:8080 -> 8080
```

![image](https://user-images.githubusercontent.com/3299086/167685251-69718cd0-a7f2-4c27-b7e5-7974f9e1c324.png)